pkg/cert/truststore.go

```diff
@@ -17,6 +17,8 @@ import (
 type Verifier interface {
 	// Verify verifies the given certificate. The validity of the certificate is checked against the given moment in time.
 	Verify(*x509.Certificate, time.Time) error
+	// VerifiedChain verifies the certificate against the truststore and returns the chain of trust as result
```

Comment:
I think it returns the chains (notice 's') of trust as it is a 2-dimensional array?

Reply:
multiple chains can apply

Reply:
This happens when certs are renewed